packages/noports_core/test/sshnp/util/sshnp_initial_tunnel_handler/sshnp_openssh_initial_tunnel_handler_mocks.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:mocktail/mocktail.dart';
+import 'package:noports_core/src/common/io_types.dart';
+import 'package:noports_core/sshnp_foundation.dart';
+
+import '../../sshnp_mocks.dart';
+
+/// Function Stubbing
+abstract class StartInitialTunnelCaller {
+  void call();
+}
+
+class StartInitialTunnelStub extends Mock implements StartInitialTunnelCaller {}
+
+/// Stubbed Mixin that we are testing
+mixin StubbedSshnpOpensshInitialTunnelHandler
+    on SshnpOpensshInitialTunnelHandler {
+  late StartInitialTunnelStub _stubbedStartInitialTunnel;
+  late StartProcessStub _stubbedStartProcess;
+
+  void stubSshnpOpensshInitialTunnelHandler({
+    required StartInitialTunnelStub stubbedStartInitialTunnel,
+    required StartProcessStub stubbedStartProcess,
+  }) {
+    _stubbedStartInitialTunnel = stubbedStartInitialTunnel;
+    _stubbedStartProcess = stubbedStartProcess;
+  }
+
+  @override
+  Future<Process?> startInitialTunnel({
+    required String identifier,
+    ProcessStarter startProcess = Process.start,
+  }) {
+    _stubbedStartInitialTunnel();
+    return super.startInitialTunnel(
+      identifier: identifier,
+      startProcess: _stubbedStartProcess.call,
+    );
+  }
+}
+
+/// Stubbed Sshnp instance with the mixin
+class StubbedSshnp extends SshnpCore
+    with
+        SshnpOpensshInitialTunnelHandler,
+        StubbedSshnpOpensshInitialTunnelHandler {
+  StubbedSshnp({
+    required super.atClient,
+    required super.params,
+    required SshnpdChannel sshnpdChannel,
+    required SshrvdChannel sshrvdChannel,
+  })  : _sshnpdChannel = sshnpdChannel,
+        _sshrvdChannel = sshrvdChannel;
+
+  @override
+  AtSshKeyPair? get identityKeyPair => throw UnimplementedError();
+
+  @override
+  AtSshKeyUtil get keyUtil => throw UnimplementedError();
+
+  @override
+  Future<SshnpResult> run() => throw UnimplementedError();
```

Comment:
run is not tested (since it is an abstract member in Core)

identityKeyPair and keyUtil are used within the implementations for run() so are not needed to be defined for this mock.